### ISSUE TITLE
Unit tests fail in AllegroGraph v6.6.0

### ISSUE THREAD

**Issue by mdorf** [ncbo/ontologies_linked_data#103]
The tests fail with the following error:
```
Object is not valid. Check errors.
/Users/mdorf/dev/ncbo/goo/lib/goo/base/resource.rb:333:in `save'
/Users/mdorf/dev/ncbo/ontologies_linked_data/lib/ontologies_linked_data/models/base.rb:13:in `save'	/Users/mdorf/dev/ncbo/ontologies_linked_data/lib/ontologies_linked_data/models/ontology.rb:404:in `save'
/Users/mdorf/dev/ncbo/ontologies_linked_data/test/security/test_access_control.rb:32:in `before_suite'
/Users/mdorf/dev/ncbo/ontologies_linked_data/test/test_case.rb:70:in `_run_suite'
```

**Comment by mdorf** [comment]
Minimal troubleshooting revealed the following issue:

The `self.before_suite` block of `test/security/test_access_control.rb` fails because the ontology object that is being saved fails the `obj.valid?` check:

**Comment by graybeal** [comment]
From Franz technical support:

Can you explain more on what is failing. I looked at the github site, but I could not figure out how to run anything myself, so I do not know how to reproduce the problem.

Are you ruuing a SPARQL query like the following:

 (sparql:run-sparql "select * { bind(true as ?x) }" :results-format  :sparql-json)

(It does not have to be run in Lisp. The important thing is the :results-format being SPARQL-JSON.) There is a bug in 6.4.2-6.6.0 (but not in 6.4.1 and fixed in 7.0.0 soon to be released) where the result is {"type":"literal","value":"true"} rather than {"type":"literal", "datatype":"http://www.w3.org/2001/XMLSchema#boolean",  "value":"true"}

That seems to be the problem you are having. The value of @summary-only is the string "false" rather than the boolean "false"^^xs:boolean.

If we could see where the value is being produced, perhaps we could suggest a workaround.

**Comment by graybeal** [comment]
Preliminary indications are that 7.0.0 takes care of this. We will confirm this once we've finished testing.
